### DISCUSSION TITLE
DevStats

### DISCUSSION THREAD

**Discussion by TracyRagan** [cdfoundation/wg-lfx-requirements#10] [General]
The Jenkins Community rely on DevStats. Want to make sure it stays in.

**Comment by MarkEWaite** [comment]
The Jenkins project uses the following devstats reports:

* [28 day moving average of contributors and the companies that employ them](https://jenkins.devstats.cd.foundation/d/7/companies-contributing-in-repository-groups?orgId=1)
* [7 day moving average of pull requests in jenkinsci and jenkins-infra repositories](https://jenkins.devstats.cd.foundation/d/15/new-prs-in-repository-groups?orgId=1&var-period=d7&var-repogroup_name=All)
* [7 day moving average of pull request time to engagement in jenkinsci and jenkins-infra repositories](https://jenkins.devstats.cd.foundation/d/10/pr-time-to-engagement?orgId=1)
* [weekly average of pull request time to engagement in documentation repository](https://jenkins.devstats.cd.foundation/d/10/pr-time-to-engagement?orgId=1&var-period=w&var-repogroup_name=jenkins-infra%2Fjenkins.io&from=now-2y&to=now)
* [7 day moving average of pull request time from open to merge in documentation repository](https://jenkins.devstats.cd.foundation/d/16/opened-to-merged?orgId=1&from=now-1y&to=now&var-period=d7&var-repogroup_name=jenkins-infra%2Fjenkins.io)
